annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/EnsemblController.java

```diff
@@ -0,0 +1,187 @@
+package org.cbioportal.genome_nexus.annotation.web;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import org.cbioportal.genome_nexus.annotation.domain.EnsemblTranscript;
+import org.cbioportal.genome_nexus.annotation.service.EnsemblService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController // shorthand for @Controller, @ResponseBody
+@CrossOrigin(origins="*") // allow all cross-domain requests
+@RequestMapping(value= "/")
+@Api(tags = "ensembl-controller", description = "Ensembl Controller")
```

Comment:
Also add `@PublicApi` annotation?

Reply:
We decided to go for internal instead, because we're not 100% happy about the endpoints yet (specifically isoform override)